### PR TITLE
content(security): add a note about rate-limiting

### PR DESCRIPTION
**Pull request by sinapirani**
The place of importing ConfigService and ConfigModule was not clear.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Docs
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by kamilmysliwiec** [comment]
cc @jmcdo29

File: content/security/rate-limiting.md [hunk 1]
```diff
@@ -232,14 +232,19 @@ One approach would be to use a factory function:
       imports: [ConfigModule],
       inject: [ConfigService],
       useFactory: (config: ConfigService) => ([{
-        ttl: config.get('THROTTLE_TTL'),
-        limit: config.get('THROTTLE_LIMIT'),
+        throttlers: [
```
Comment (Tony133) [inline content/security/rate-limiting.md:235]: I don't think this example should be modified. 
The `throttlers` property of the options object if I am not mistaken is used on custom storage, and it always wraps `ttl` and `limit` in an array, perhaps a separate code snippet should be created for this case. Right @jmcdo29  ? 😄

Comment (sinapirani) [reply content/security/rate-limiting.md:235]: @Tony133 It was my mistake. You are right. :)

Comment (Tony133) [reply content/security/rate-limiting.md:235]: @sinapirani, no problem 😉👍, at most edit this PR by adding a small section dedicated to the option: `throttlers` if you think it should be documented better or create a new PR, but maybe first ask @jmcdo29 how he wants to document this part 😉, I know he was working on it these weeks 🙂

**Pull request closed without merging**